File: app/src/main/java/valiev/timur/tapfight/presentation/startGame/StartGameActivity.java
package valiev.timur.tapfight.presentation.startGame;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import valiev.timur.tapfight.R;
import valiev.timur.tapfight.presentation.game.view.GameActivity;

public class StartGameActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_start_game);
    }

    public void startGame(View view) {
        startActivity(new Intent(this, GameActivity.class));
    }
}

File: app/src/main/java/valiev/timur/tapfight/presentation/startGame/StartGameActivity.kt
package valiev.timur.tapfight.presentation.startGame

import android.content.Context
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_start_game.*
import valiev.timur.tapfight.R
import valiev.timur.tapfight.presentation.game.view.GameActivity

class StartGameActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_start_game)

        start_game_button.setOnClickListener {
            startActivity(GameActivity.createIntent(this))
        }
    }

    companion object {

        fun createIntent(context: Context) = Intent(context, StartGameActivity::class.java)
    }
}